server_installation/topics/operator/keycloak-cr.adoc

```diff
@@ -0,0 +1,67 @@
+
+=== Keycloak Custom Resource
+
+The {project_operator} allows users to manage {project_name} clusters by using Keycloak Custom Resources:
+
+.`Keycloak` Custom Resource
+```yaml
+apiVersion: keycloak.org/v1alpha1
+kind: Keycloak
+metadata:
+  name: example-keycloak
+  labels:
+    app: sso
+spec:
+  instances: 1
+  externalAccess:
+    enabled: True
+```
+
+The `Spec` contains three objects (two of them are listed the above and the external database has been covered in the other paragraphs of this manual):
+
+* `instances` - controls the number of instances running in HA mode
+* `externalAccess` - if the `enabled` flag is set to `True`, {project_operator} will create a Route or an Ingress (depending on available types in the cluster) for {project_name} cluster.
```

Comment:
Maybe "if the `enabled` flag is set to `True`, depending on the types available in the cluster {project_operator} will create a Route or an Ingress for {project_name} cluster."?

Reply:
Fixed